wikidict/__main__.py

```diff
@@ -23,8 +23,11 @@
                                 - "data/$LOCALE/dicthtml-$LOCALE.zip": Kobo format.
                                 - "data/$LOCALE/dict-$LOCALE.df": DictFile format.
   --find-templates          DEBUG: Find all templates in use.
-  --check-random-words=N    Get and render N words.
-                            Then compare with the rendering done on the Wiktionary to catch errors.
+  --check-words             Render words, then compare with the rendering done on the Wiktionary to catch errors.
+  --random                  randomly if --random
+  --count=N                 If -1 check all words [default: 100]
+  --offset=M                Offset will remove words before starting.
+  --input=FILENAME          A list of words, one by line
```

Comment:
This looks ugly, no?

To be right, those options should be suboptions to `--check-words`, something like:
```suggestion
  --check-words             Render words, then compare with the rendering done on the Wiktionary to catch errors.

                            --random          Check random words.
                            --count=N         If -1 check all words, else check N words [default: 100]
                            --offset=M        Start checking words at offset M if M is a number, else start checking words at word M.
                            --input=FILENAME  A list of words, one by line.
```

I am still not satisfied with those changes though. :thinking:

Reply:
Yes, it's ugly, but I couldn't make it work differently :-/ Your version looks better.